### PR TITLE
Fix NPE when using MessageDigest

### PR DESCRIPTION
**Pull request by CarstenHollmann**
- Fix NPE because the static init method was removed where the MessageDigest was initialized.
- Change algorithm
- Add simple test

### CODE REVIEW

**Comment by autermann** [comment]
Better create a static initializer block throwing a `Error` when a `NoSuchAlgorithmException` is thrown... This approach is not thread safe and will throw a NPE on failure...

**Comment by EHJ-52n** [comment]
In addition, it might be good to be able override the algorithm by configuration value.

...and the log statement it stating SHA1 but SHA256 is used... go for a variable here...

**Comment by nuest** [comment]
I think this should not be merged, too.

**Comment by CarstenHollmann** [comment]
@autermann the static initializer block is implemented as as before he was removed.

File: core/src/main/java/org/n52/iceland/util/JavaHelper.java [hunk 1]
```diff
@@ -44,12 +49,25 @@
...
 
+
+    /**
+     * Instantiation of the message digest
+     */
+    static {
+        try {
+            messageDigest = MessageDigest.getInstance("SHA-256");
+        } catch (final NoSuchAlgorithmException nsae) {
+            LOGGER.error("Error while getting SHA-256 messagedigest!", nsae);
```
Comment (autermann) [inline core/src/main/java/org/n52/iceland/util/JavaHelper.java]: Why logging? Just throw an `Error`...

**Pull request merged**